GliaWidgets/View/Chat/Entry/ChatMessageEntryStyle.swift

```diff
@@ -8,8 +8,11 @@ public struct ChatMessageEntryStyle {
     /// Color of the message text.
     public var messageColor: UIColor
 
-    /// Placeholder text of the message text view.
+    /// Placeholder text used when user is engaged.
     public var placeholder: String
+    
+    /// Placeholder text used when user input is required to start engagement.
```

Comment:
Kairi (Glia's tech writer) really likes the explanations to be as detailed as possible, so I'd leave the `of the message text view part` (or even better `message text **input** view`) and just add `used when **the** user is engaged` (notice an extra 'the') / `when user input is required to start engagement` to the end.